examples/go/httpingress/httpingress.go

```diff
@@ -19,15 +19,17 @@ type GetResponse struct {
 	Message string `json:"message"`
 }
 
+// Example: curl -i http://localhost:8892/ingress/http/users/123/posts?postId=456
+//
 //ftl:verb
-//ftl:ingress http GET /http/users/{userID}/posts/{postID}
+//ftl:ingress http GET /http/users/{userID}/posts
 func Get(ctx context.Context, req builtin.HttpRequest[GetRequest]) (builtin.HttpResponse[GetResponse], error) {
 	logger := ftl.LoggerFromContext(ctx)
 	logger.Infof("Path: %s", req.Path)
 	logger.Infof("Method: %s", req.Method)
-	logger.Infof("Query: %s", req.Query)
-	logger.Infof("Body: %s", req.Body)
-	logger.Infof("Headers: %s", req.Headers)
+	logger.Infof("Query: %v", req.Query)
+	logger.Infof("Body: %v", req.Body)
+	logger.Infof("Headers: %v", req.Headers)
```

Comment:
Maybe this should be debug?

Reply:
Yeah I was mostly just using them to make testing this httpIngress stuff super easy and show the logger, but none of this is actually needed :)